### PR TITLE
dev

### PR DESCRIPTION
**Pull request by denduluriramasaimohanakhil**
coverage:85%
 lint:53
RemovingLint:
 coverage:85%
 lint:0

### CODE REVIEW

File: src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java [hunk 1]
```diff
@@ -163,24 +153,13 @@ public FormDataResponse saveFormData(FormDataSchema formDataSchema) throws IOExc
                             this.formDataAuditService.saveFormDataAudit(formDataAuditSchema);
                         }
                     }
-                    if (!documentFlag)
-                    {
-                        throw new InvalidInputException(FORM_DATA_NOT_FOUND_WITH_GIVEN_FORMDATAID_IN_MONGO_AND_ELASTIC,globalMessageSource.get(FORM_DATA_NOT_FOUND_WITH_GIVEN_FORMDATAID_IN_MONGO_AND_ELASTIC,uniqueDocumentId));
-                    }
+                    checkDocumentFlag(documentFlag, uniqueDocumentId);
```
Comment (VaibhavTechsophy) [inline src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java]: what is the check for?

File: src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java [hunk 2]
```diff
@@ -88,25 +88,18 @@ public FormDataResponse saveFormData(FormDataSchema formDataSchema) throws IOExc
         Integer version = null;
```
Comment (VaibhavTechsophy) [inline src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java]: Need to understand what this code is doing, please explain, it looks unnecessarily complex

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]
Need to fix duplicate code, and some parts seem overly complex please discuss with me

File: src/main/java/com/techsophy/tsf/runtime/form/service/impl/SequenceGeneratorServiceImpl.java [hunk 3]
```diff
@@ -27,7 +26,7 @@ public SequenceGeneratorResponse generateSequence(SequenceGeneratorDTO idGenerat
         int length= Integer.parseInt((idGeneratorDTO.getLength()));
         SequenceGeneratorDefinition existingDefinition;
         SequenceGeneratorDefinition sequenceGeneratorDefinition=new SequenceGeneratorDefinition();
-        sequenceGeneratorDefinition.setUpdatedOn(Instant.now());
+        sequenceGeneratorDefinition.setUpdatedOn(String.valueOf(Date.from(Instant.now())));
```
Comment (VaibhavTechsophy) [inline src/main/java/com/techsophy/tsf/runtime/form/service/impl/SequenceGeneratorServiceImpl.java:29]: what is the issue here?

**Comment by denduluriramasaimohanakhil** [comment]
Vaibhav can u please merge now all sonar checks have passed and code coverage is at 85% and sonar lint issues reduced to ZERO

**Review by VaibhavTechsophy — APPROVED** [review]

**Pull request merged**